test/extended/router/scoped.go

```diff
@@ -28,7 +28,7 @@ import (
 
 const changeTimeoutSeconds = 3 * 60
 
-var _ = g.Describe("[sig-network][Feature:Router][apigroup:route.openshift.io][apigroup:template.openshift.io]", func() {
+var _ = g.Describe("[sig-network][Feature:Router][apigroup:route.openshift.io]", func() {
```

Comment:
The `BeforeEach` block still uses the `router-common.yaml` fixture on line 61, so I think "[apigroup:template.openshift.io]" should still be here in this commit.  It isn't too important though if a subsequent commit removes the use of the fixture.